src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -160,6 +159,16 @@ class JooqBurdenEstimateRepository(
                 ?: throw UnknownObjectError(setId, "burden-estimate-set")
     }
 
+    override fun getBurdenEstimateSet(groupId: String, touchstoneVersionId: String, scenarioId: String,
+                                      burdenEstimateSetId: Int): BurdenEstimateSet
+    {
+        //Get burden estimate set with validation
+        //Confirm that this burden estimate set belongs to this group, touchstone and scenario
+        checkBurdenEstimateSetPathParameters(groupId, touchstoneVersionId, scenarioId, burdenEstimateSetId)
```

Comment:
Could also then just throw the 
```
                .and(RESPONSIBILITY_SET.MODELLING_GROUP.eq(groupId))
                .and(RESPONSIBILITY_SET.TOUCHSTONE.eq(touchstoneVersionId))
                .and(SCENARIO_DESCRIPTION.ID.eq(scenarioId))
```
clauses into the existing select statement, rather than doing 2 queries.